include/proto.h

```diff
@@ -491,6 +491,16 @@ enum {
    SL_FLICKER_OFF      = 2,         // Turn flickering off
 };
 
+/* Item text color types */
+typedef enum {
```

Comment:
I believe you can actually leave out the "typedef" in C++, give it a try.

Reply:
I tried but if I remove the typedef I have to use `enum` every time I declare a variable.

example without `typedef`:
`enum item_text_color text_color_value= ITEM_TEXT_COLOR_NORMAL;`

example with `typedef`:
`item_text_color text_color_value= ITEM_TEXT_COLOR_NORMAL;`

I might be wrong, but if the above is true then I think I would recommend sticking with a `typedef` declaration so the rest of the code is a bit cleaner when using `item_text_color`